addons/board/static/src/board_controller.scss

```diff
@@ -27,7 +27,6 @@
 }
 
 .o-dashboard-action {
-    background-color: white;
```

Comment:
No problem with the removal of the `background-color` as our dark mode doesn't manage hard coded color values like this one.

But I think we still need to set a `background-color` to `o-dashboard-action` or it will be the same value as the background of the page and it will not be consistent with what we have in light mode (grey background for the page and white background for `o-dashboard-action`). I suggest you to add a `bg-view` class to the `o-dashboard-action` `div` in `board_controller.xml`